app/src/main/java/com/mediapipe/example/sign_language/HolisticLandmarkerHelper.kt

```diff
@@ -0,0 +1,189 @@
+package com.mediapipe.example.sign_language
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Matrix
+import android.os.SystemClock
+import android.util.Log
+import androidx.camera.core.ImageProxy
+import com.google.mediapipe.framework.image.BitmapImageBuilder
+import com.google.mediapipe.framework.image.MPImage
+import com.google.mediapipe.tasks.core.BaseOptions
+import com.google.mediapipe.tasks.core.Delegate
+import com.google.mediapipe.tasks.vision.core.RunningMode
+import com.google.mediapipe.tasks.vision.holisticlandmarker.HolisticLandmarker
+import com.google.mediapipe.tasks.vision.holisticlandmarker.HolisticLandmarkerResult
+
+class HolisticLandmarkerHelper(
+    private var runningMode: RunningMode = RunningMode.IMAGE, private val context: Context,
+    val minFacePresenceConfidence: Float,
+    val minHandLandmarksConfidence: Float,
+    val minPosePresenceConfidence: Float,
+    val minFaceDetectionConfidence: Float,
+    val minPoseDetectionConfidence: Float,
+    val minFaceSuppressionThreshold: Float,
+    val minPoseSuppressionThreshold: Float,
+    // this listener is only used when running in RunningMode.LIVE_STREAM
+    private val landmarkerHelperListener: LandmarkerListener? = null
+) {
+
+    companion object {
+        private const val MP_HOLISTIC_LANDMARKER_TASK =
+            "holistic_landmarker.task"
+        const val TAG = "HolisticLandmarkerHelper"
+        const val OTHER_ERROR = 0
+        const val DEFAULT_MIN_FACE_PRESENCE_CONFIDENCE = 0.5F
+        const val DEFAULT_MIN_HAND_LANDMARKS_CONFIDENCE = 0.5F
+        const val DEFAULT_MIN_POSE_PRESENCE_CONFIDENCE = 0F
+        const val DEFAULT_MIN_FACE_DETECTION_CONFIDENCE = 0.5F
+        const val DEFAULT_MIN_POSE_DETECTION_CONFIDENCE = 0F
+        const val DEFAULT_MIN_FACE_SUPPRESSION_THRESHOLD = 0.5F
+        const val DEFAULT_MIN_POSE_SUPPRESSION_THRESHOLD = 0F
+
+        const val MIN_CONFIDENCE = 0F
```

Comment:
Let's set min confidence to 0.3 and max to 0.9.

Reply:
Also let's set those min/max values from the fragment (like here: https://github.com/googlesamples/mediapipe/blob/main/examples/face_landmarker/android/app/src/main/java/com/google/mediapipe/examples/facelandmarker/fragment/CameraFragment.kt#L191) so we can change the value easily for each different category.